db/sysinfo.go

```diff
@@ -91,15 +92,16 @@ func TopSysinfoFields() ([]TopSysinfo, error) {
 	rows, err := db.Query(context.Background(), `
 	SELECT fields.field, fields.name, FLOOR((fields.count::float / fields.total_count::float) * 100) FROM (
 		SELECT 1 as order, * FROM top_field('Distro')
-		UNION SELECT 2 as order, * FROM top_field('DeWm')
-		UNION SELECT 3 as order, * FROM top_field('DisplayProtocol')
-		UNION SELECT 4 as order, * FROM top_field('Shell')
-		UNION SELECT 5 as order, * FROM top_field('Terminal')
-		UNION SELECT 6 as order, * FROM top_field('Bar')
-		UNION SELECT 7 as order, * FROM top_field('Gtk3Theme')
-		UNION SELECT 8 as order, * FROM top_field('GtkIconTheme')
-		UNION SELECT 9 as order, * FROM top_field('Editor')
-		UNION SELECT 10 as order, * FROM top_field('Cpu')
+	    UNION SELECT 2 as order, * FROM top_field('Host')
+        UNION SELECT 3 as order, * FROM top_field('DeWm')
```

Comment:
Looks like you broke the indenting on these lines

Reply:
oof yes :(